test/css/component_specific_selectors_test.rb

```diff
@@ -39,6 +39,16 @@ class ComponentSpecificSelectorsTest < Minitest::Test
       ".tabnav-extra",
       ".tabnav-btn"
     ],
+    Primer::Alpha::TextField => [
+      ".FormControl-inlineValidation p",
+      ".FormControl-select",
+      ".FormControl-textarea",
+      ".FormControl-input",
+      ".FormControl-input-wrap",
+      ".FormControl-select-wrap",
+      ".FormControl-checkbox-wrap",
+      ".FormControl-radio-wrap"
```

Comment:
Currently all `.FormControl` styles are part of `Primer::Alpha::TextField` and located at `app/components/primer/alpha/text_field.pcss`. But the `.FormControl` styles are not just for text fields and also include styles for select, radio, checkbox etc. so not sure if that should be split up?

Or we could also move `.FormControl` to a place where we have styles that get shared across multiple components? Maybe [app/lib/primer/css/](https://github.com/primer/view_components/tree/main/app/lib/primer/css).